deploy/test/helm-props/k8s/helm-app.yaml

```diff
@@ -0,0 +1,29 @@
+# This app is created to run end-to-end with the Secrets Provider Job
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: test-env{{ ID }}
+  name: test-env{{ ID }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test-env{{ ID }}
+  template:
+    metadata:
+      labels:
+        app: test-env{{ ID }}
+    spec:
+      serviceAccountName:  {{ SERVICE_ACCOUNT }}
+      containers:
+      - image: centos:7
+        name: test-app{{ ID }}
+        command: ["sleep"]
+        args: ["infinity"]
+        env:
+        - name: TEST_SECRET{{ ID }}
```

Comment:
why do we need the ID here? it seems that we always have only one secret in the env so it can be just `TEST_SECRET`

Reply:
Not for test `TEST_ID_18_helm_multiple_provider_multiple_secrets.sh` so I wanted this to be configurable